intranet/apps/announcements/forms.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, *args, **kwargs):
         )
 
     expiration_date = forms.DateTimeInput()
-    # notify_email_all = forms.BooleanField(required=False, label="Send Email to All")
+    notify_email_all = forms.BooleanField(required=False, label="Send Email to All")
```

Comment:
Does this field actually do anything?

Reply:
It sets this field in the model: https://github.com/tjcsl/ion/blob/c0e0b834d923d61811ec5c3763b33d6ae249d7be/intranet/apps/announcements/models.py#L112

Which is checked here: https://github.com/tjcsl/ion/blob/34a2e0c1b4e06433165b4b1b90b32b84c5c5bb0f/intranet/apps/announcements/views.py#L48

And gets passed to this function, which uses it to determine which users to send the email to: https://github.com/tjcsl/ion/blob/34a2e0c1b4e06433165b4b1b90b32b84c5c5bb0f/intranet/apps/announcements/notifications.py#L118

Like [I said in #601](https://github.com/tjcsl/ion/issues/601#issuecomment-517966217), the functionality is all there. It was just disabled.